clippy_lints/src/manual_let_else.rs

```diff
@@ -358,37 +396,59 @@ fn pat_allowed_for_else(cx: &LateContext<'_>, pat: &'_ Pat<'_>, check_types: boo
     !has_disallowed
 }
 
-/// Checks if the passed block is a simple identity referring to bindings created by the pattern
-fn expr_is_simple_identity(pat: &'_ Pat<'_>, expr: &'_ Expr<'_>) -> bool {
+/// Checks if the passed block is a simple identity referring to bindings created by the pattern,
```

Comment:
nit: might be worth including an example here

Reply:
There is an example just below in the body of the function, should I move it?

Reply:
Yep!